src/mbed_tools/project/mbed_program.py

```diff
@@ -6,18 +6,16 @@
 import logging
 
 from pathlib import Path
-from typing import List, Dict
+from typing import Dict
```

Comment:
In commit "MbedProgram: Remove library handling", please explain the motivation behind the removal (based on the changes contributed in previous commits).

Reply:
Added an explanation to the commit message.